tools/importkey/README.MD

```diff
@@ -27,18 +27,20 @@ When importing a key to the key vault, a release policy is coupled with the key.
         "label": "Model Encryption Key"
     },
     "claims":[
-        {
-            "claim":  "x-ms-sevsnpvm-hostdata",
-            "equals": "99e1abd344fc0989288c44fde8de3b2a248b1b95814df8955d0c305a7db46680"
-        },
-        {
-            "claim": "x-ms-compliance-status",
-            "equals": "azure-compliant-uvm"
-        },
-        {
-            "claim": "x-ms-sevsnpvm-is-debuggable",
-            "equals": "false"
-        }
+        [
```

Comment:
Do we want this included? Seems to be an indentation change

Reply:
Sorry @hgarvison I should have made my PR clearer earlier. The example in README didn't work because importkey requires claims to be an array of array